secp256k1-sys/src/lib.rs

```diff
@@ -673,6 +672,80 @@ mod fuzz_dummy {
         fn rustsecp256k1_v0_4_1_context_preallocated_clone(cx: *const Context, prealloc: *mut c_void) -> *mut Context;
     }
 
+    mod once {
```

Comment:
This needs some kind of descriptive comment describing why its here, that it's only used in fuzzing, and that we don't really care too much...................all that said, maybe we *should* care a bit more, I realized yesterday that `global-context-less-secure` relies on `std`, even though the whole point of it is to avoid relying on std (it predates no-std so that's expected, but still). It only actually relies on `std::sync::Once`, so maybe we should use this there as well. Not sure how to get it across both crates, though, maybe just make a single file and symlink them across the two crates to avoid making it public?

Reply:
In that case I think it should condinionaly use `Once` from `std`  as suggested in #352

Reply:
The whole point of the `global-context-less-secure` feature is to work on `no-std`, and sadly `Once` is not `no-std` as it requires thread-parking support.

Reply:
Since cargo features are additive it's still useful to avoid performance issues if two crates turn on these features. Although maybe they should've been `--cfg` parameters, not features...

Reply:
Oh, sorry, I misread your comment again lol. Yes, conditionally using `Once` if `std` is turned on is fine, but we still need to somehow have this `once` available in `secp256k1` if we want to fix `global-context-less-secure` not building with `no-std`.